### PR TITLE
Add data summary overview to docs

### PR DESCRIPTION
**Pull request by ghop02**
This creates a page on the docs site that gives an overview of all the available data.  

I realized there was no good place to share all of the data that we include in the API, so wrote a script that lets us generate an overview of what's in the api

### CODE REVIEW

File: api/data_overview_builder.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
+"""
+Data Overiew Builder generates a markdown file with information about data in the API.
+
+By default this is shown on the docs site.
+
```
Comment (TomGoBravo) [inline api/data_overview_builder.py:5]: how and when is this run? How does someone running it check and deploy the output?

File: api/data_overview_builder.py [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+    name: str
+    fields: List[pydantic.Field]
+    description: str = ""
+
+
+CASE_DESCRIPTION = """
+
+
+"""
```
Comment (TomGoBravo) [inline api/data_overview_builder.py:55]: WIP? CASE_DESCRIPTION is empty and will be more readable with new lines after it.

File: api/data_overview_builder.py [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+SECTIONS = [
+    Section("Cases", CASES_FIELDS),
```
Comment (TomGoBravo) [inline api/data_overview_builder.py:57]: Fields type looks wrong. `List[pydantic.Field]` vs `Fields`.

**Review by TomGoBravo — CHANGES_REQUESTED** [review]

File: api/data_overview_builder.py (hunk 1 above)
Comment (ghop02) [reply api/data_overview_builder.py:5]: This is run when you run `./run.py api update-schemas`.  You can check the output by running the docs locally and it's deployed through the normal docs deploy process `tools/deploy-docs.sh` (I do realize this is not obvious or really stated anywhere, will try to add to readme

File: api/docs/docs/data.md [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+
+  New confirmed or suspected cases.
+
+New cases are a processed timeseries of cases - summing new cases may not equal
+the cumulative case count.
+
+Processing steps:
+ 1. If a region does not report cases for a period of time, the first day
+    cases start reporting again will not be included. This day likely includes
+    multiple days worth of cases and can be misleading to the overall series.
```
Comment (tommacmullen) [inline api/docs/docs/data.md:29]: Alternatively: "If a region does not report cases for a period of time but then begins reporting again, we will exclude the first day that reporting recommences. This first day likely includes multiple days worth of cases and can be misleading to the overall series."

File: api/docs/docs/data.md [hunk 5]
```diff
@@ -0,0 +1,210 @@
...
+ 2. Any days with negative new cases are removed.
```
Comment (tommacmullen) [inline api/docs/docs/data.md:30]: Alt: "We remove any days with negative new cases"

File: api/docs/docs/data.md [hunk 6]
```diff
@@ -0,0 +1,210 @@
...
+ 3. An outlier detection filter is applied to the timeseries, removing any data points that
+    seem improbable given recent case numbers.  Many times this is due to a backfill of
+    previously unreported cases.
```
Comment (tommacmullen) [inline api/docs/docs/data.md:33]: Alt: "We apply an outlier detection filter to the timeseries, which removes any data points that seem improbable given recent numbers. Many times this is due to backfill of previously unreported cases."

File: api/docs/docs/data.md [hunk 7]
```diff
@@ -0,0 +1,210 @@
...
+
+  The number of cases per 100k population calculated using a 7-day rolling average.
+
+**Where to access**  
+* CSV column names: ``metrics.caseDensity``
+* JSON file fields: ``metrics.caseDensity``, ``metricsTimeseries.*.caseDensity``
+
+### Infection Rate
+
+  R_t, or the estimated number of infections arising from a typical case.
```
Comment (tommacmullen) [inline api/docs/docs/data.md:49]: Should we either explain the computation process here or link to the doc that lays out the full computation process?

File: api/docs/docs/data.md [hunk 8]
```diff
@@ -0,0 +1,210 @@
...
+
+**Where to access**  
+* CSV column names: ``metrics.infectionRate``
+* JSON file fields: ``metrics.infectionRate``, ``metricsTimeseries.*.infectionRate``
+
+### Infection Rate Ci90
+
+  90th percentile confidence interval upper endpoint of the infection rate.
```
Comment (tommacmullen) [inline api/docs/docs/data.md:57]: Do we need to explain what differentiates this from the  other infection rate mentioned above or would this be intuitive to someone working with the API? I personally don't know what IR Ci90 really means.

File: api/docs/docs/data.md [hunk 9]
```diff
@@ -0,0 +1,210 @@
...
+
+### Test Positivity Ratio
+
+  Ratio of people who test positive calculated using a 7-day rolling average.
+
+**Where to access**  
+* CSV column names: ``metrics.testPositivityRatio``
+* JSON file fields: ``metrics.testPositivityRatio``, ``metricsTimeseries.*.testPositivityRatio``
+
+### Test Positivity Ratio Details
```
Comment (tommacmullen) [inline api/docs/docs/data.md:91]: What does this query in comparison to other things mentioned?

File: api/docs/docs/data.md [hunk 10]
```diff
@@ -0,0 +1,210 @@
...
+
+**Where to access**  
+* CSV column names: ``metrics.testPositivityRatioDetails``
+* JSON file fields: ``metrics.testPositivityRatioDetails``, ``metricsTimeseries.*.testPositivityRatioDetails``
+
+
+
+## Hospitalizations
+
+### Icu Beds
```
Comment (tommacmullen) [inline api/docs/docs/data.md:103]: capitalize? "ICU"

File: api/docs/docs/data.md [hunk 11]
```diff
@@ -0,0 +1,210 @@
...
+* CSV column names: ``metrics.icuCapacityRatio``
+* JSON file fields: ``metrics.icuCapacityRatio``, ``metricsTimeseries.*.icuCapacityRatio``
+
+### Icu Headroom Ratio
+
+  
+
+**Where to access**  
+* CSV column names: ``metrics.icuHeadroomRatio``
+* JSON file fields: ``metrics.icuHeadroomRatio``, ``metricsTimeseries.*.icuHeadroomRatio``
```
Comment (tommacmullen) [inline api/docs/docs/data.md:145]: Wondering whether it makes sense to keep all ICU stuff together and move this up. I understand how it's currently organized (bed counts first and then the the ratios), but maybe worth considering the change.

File: api/docs/docs/data.md [hunk 12]
```diff
@@ -0,0 +1,210 @@
...
+
+### Hospital Beds
+
+  Information about acute bed utilization details.
+
+Fields:
+ * capacity - Current staffed acute bed capacity.
+ * currentUsageTotal - Total number of acute beds currently in use
+ * currentUsageCovid - Number of acute beds currently in use by COVID patients.
+ * typicalUsageRate - Typical acute bed utilization rate.
```
Comment (tommacmullen) [inline api/docs/docs/data.md:125]: Seeing the "fields" section here made me wonder whether we should have an "inputs" section. Like for all the topline metrics do we want to show the inputs we use for each? 
 
e.g. ICU Capacity Ratio

Inputs:

- capacity - current staffed ICU bed capacity
- currentUsagetotal - etc.

File: api/data_overview_builder.py (hunk 1 above)
Comment (TomGoBravo) [reply api/data_overview_builder.py:5]: Thank you for adding to the README :-)

**Review by TomGoBravo — APPROVED** [review]
Looks good from engr side. Please address tommacmullen's concerns too.

**Pull request merged**